pages/common/cradle.md

```diff
@@ -0,0 +1,20 @@
+# cradle
+
+> The Cradle PHP framework.
+> See `cradle-install`, `cradle-deploy` and other pages for additional information.
+
+- Connect to a server:
+
+`cradle connect {{server_name}}`
+
+- Call general help:
```

Comment:
`Call` sounds rather odd here. What about `Show` instead? Same for the next example down.

Reply:
Yeah, I don't normally add the `help` commands to tldr pages. But I was looking at `git` for the format.